frontend/common/src/constants/localizedConstants.tsx

```diff
@@ -119,6 +119,55 @@ export const workRegions = defineMessages({
   },
 });
 
+export const workRegionsDetailed = defineMessages({
+  [WorkRegion.Telework]: {
+    defaultMessage:
+      "<bold>Virtual: </bold> Work from home, anywhere in Canada.",
+    description: "The work region of Canada described as Telework.",
+  },
+  [WorkRegion.NationalCapital]: {
+    defaultMessage:
+      "<bold>National Capital Region:</bold> Ottawa, ON and Gatineau, QC.",
+    description: "The work region of Canada described as National Capital.",
+  },
+  [WorkRegion.Atlantic]: {
+    defaultMessage:
+      "<bold>Atlantic Region: </bold> New Brunswick, Newfoundland and Labrador, Nova Scotia and Prince Edward Island.",
+    description: "The work region of Canada described as Atlantic.",
+  },
+  [WorkRegion.Quebec]: {
+    defaultMessage: "<bold>Quebec Region: </bold>excluding Gatineau.",
+    description: "The work region of Canada described as Quebec.",
+  },
+  [WorkRegion.Ontario]: {
+    defaultMessage: "<bold>Ontario Region:</bold> excluding Ottawa.",
+    description: "The work region of Canada described as Ontario.",
+  },
+  [WorkRegion.Prairie]: {
+    defaultMessage:
+      "<bold> Prairie Region: </bold> Manitoba, Saskatchewan, Alberta.",
+    description: "The work region of Canada described as Prairie.",
+  },
+  [WorkRegion.BritishColumbia]: {
+    defaultMessage: "<bold>British Columbia Region </bold>",
+    description: "The work region of Canada described as British Columbia.",
+  },
+  [WorkRegion.North]: {
+    defaultMessage:
+      "<bold>North Region:</bold> Yukon, Northwest Territories and Nunavut.",
+    description: "The work region of Canada described as North.",
+  },
+});
+
+export const getworkRegionsDetailed = (
```

Comment:
Small typo, should be getWorkRegionsDetails with captial W

Reply:
Ok. Thank you! Done.